Cargo.toml

```diff
@@ -25,7 +25,7 @@ failure = "0.1.5"
 flat-tree = "4.1.0"
 lazy_static = "1.3.0"
 memory-pager = "0.9.0"
-merkle-tree-stream = "0.10.0"
+merkle-tree-stream = { version = "0.10.0", git = "https://github.com/bltavares/merkle-tree-stream", branch = "send" }
```

Comment:
I believe we've merged this, but not published. Added you as a contrib, let me know if you need publish access on crates.io as well!

Reply:
New version released.